setup.py

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env python
+
+from setuptools import setup
+from catkin_pkg.python_setup import generate_distutils_setup
+
+d = generate_distutils_setup(
+    # #  don't do this unless you want a globally visible script
+    # scripts=['bin/myscript'],
+    packages=["ed_py"],
```

Comment:
I still think this abbreviated name is inconsistent, not worth saving those 4 chars.

Reply:
I also agree, our codebase can handle the 4 extra chars.